codex-rs/core/src/config.rs

```diff
@@ -1307,8 +1320,14 @@ exclude_slash_tmp = true
         let loaded = load_global_mcp_servers(codex_home.path())?;
         assert_eq!(loaded.len(), 1);
         let docs = loaded.get("docs").expect("docs entry");
-        assert_eq!(docs.command, "echo");
-        assert_eq!(docs.args, vec!["hello".to_string()]);
+        match &docs.transport {
```

Comment:
I really want these tests to show the before/after TOML itself to ensure we are creating files the right way. We didn't do a great job with trusted projects originally, for example.